core/jobs/batch_jobs/exp_validation_jobs.py

```diff
@@ -71,8 +71,10 @@ def get_invalid_links(self, dictionary: Dict[str, str]) -> List[str]:
         cleaned_links = []
         for link in links:
             # Remove &quot; from the links.
-            cleaned_links.append(
-                link.get('url-with-value').replace('&quot;', ''))
+            link_text = link.get('url-with-value')
+            if link_text is not None:
```

Comment:
Is it fine if the link is `None` or should we output an error?

Reply:
I think it would be fine if a link doesn't exist in the oppia-non-interactive-link component.